C/guide/ch_reports.xml

```diff
@@ -304,6 +304,67 @@
         </para>
       </sect3>
 
+      <sect3 id="rpt_investlots">
+        <title>Investment Lots</title>
+
+        <figure id="fig-investlots" pgwide="1">
+          <title>The Investment Lots Report</title>
+          <screenshot id="InvestmentLotsScreenShot">
+            <screeninfo>1720x982</screeninfo>
+            <mediaobject>
+              <imageobject role="html">
+                <imagedata fileref="figures/Report_Investment_Lots.png"
+                            srccredit="Brent McBride" width="&img-w;"/>
+              </imageobject>
+
+              <imageobject role="fo">
+                <imagedata fileref="figures/Report_Investment_Lots.png"
+                            srccredit="Brent McBride"/>
+              </imageobject>
+            </mediaobject>
+          </screenshot>
+        </figure>
+
+        <para>The Investment Lots report shows details about lots in a selected set of accounts of type
+        <emphasis>Stock</emphasis> or <emphasis>Mutual Fund</emphasis> (see <xref linkend="invest-sellLots" />).
+        It includes an optional capital gains summary chart and a table with columns for dates, purchases, sales, end
+        balances, capital gains (realized and unrealized, short term and long term), and return on investment (ROI).
+        It also performs optional lot validation checks.
+        </para>
+
+        <para>The report can help answer questions such as:
+        </para>
+
+        <itemizedlist>
+          <listitem>
+            <para>Which lots are best candidates for tax loss/gain harvesting?
+            </para>
+          </listitem>
+
+          <listitem>
+            <para>What capital gains have been realized for a particular time period (i.e. this year, last year,
+            etc)? Are they long term or short term gains?
+            </para>
+          </listitem>
+
+          <listitem>
+            <para>Which lots held longer than 1+ years have the largest unrealized gains, and would therefore be
+            most advantageous (tax-wise) if donated to charity?
+            </para>
+          </listitem>
+
+          <listitem>
+            <para>Are the lots (and related accounts and splits) well-formed?
+            </para>
+          </listitem>
+        </itemizedlist>
+
+        <para>For more details, see the
+          <ulink url="&url-docs;/C/gnucash-help/report-classes.html#investment-lots-report">Investment Lots</ulink>
```

Comment:
1. Please use here and later the entities from [docbook/gnc-docbookx.dtd#L119-L124](https://github.com/Gnucash/gnucash-docs/blob/master/docbook/gnc-docbookx.dtd#L119-L124)
2. `help` should be called `manual` now.